doc/source/release/1.22.3-notes.rst

```diff
@@ -5,9 +5,44 @@ NumPy 1.22.3 Release Notes
 ==========================
 
 The NumPy 1.22.3 is maintenance release that fixes bugs discovered after the
-1.22.2 release. Notable fixes are:
+1.22.2 release. The most noticeable fixes may be those for DLPack. One that may
+cause some problems is disallowing strings as inputs to logical ufuncs. It is
+still undecided how strings should be treated in those functions and it was
+thought best to simply disallow them until a decision was reached. That should
+not cause problems with older code.
 
+The Python versions supported for this release are 3.8-3.10. Note that the Mac
+wheels are now based on OS X 10.14 rather than 10.6 that was used in previous
+NumPy release cycles. 10.14 is the latest release supported by Apple.
```

Comment:
Not sure that "latest" is unambiguous here, perhaps:
```suggestion
NumPy release cycles. 10.14 is the oldest release still supported by Apple.
```

Reply:
Good point.